docs/source/using/index.adoc

```diff
@@ -1,14 +1,12 @@
-[[using-index]]
+[[using-minishift]]
 = Using Minishift
```

Comment:
Maybe change this to "Managing Minishift", now that we separated the OpenShift topics? I'm not terribly stubborn about this suggestion, btw.